### PR TITLE
added sine functionality and unit tests for sine

### PR DESCRIPTION
**Pull request by jkoors**
@GregHilston let me know if this works better.

### CODE REVIEW

**Comment by GregHilston** [comment]
Will do! I'll get around to this today and see if everything checks out and get you some feedback.

**Comment by GregHilston** [comment]
@jkoors Hey sorry for getting back to you so late, I was finishing up my semester.

I checked your PR and it looks great. Just for a little bit more practice, why don't you change the variable name `x` in the function 

`def sin(self, x)`

to `xInRadians`, just so you're sure you know how to add a new commit to an active PR.

(No need to submit another PR, committing and pushing to this branch will update the PR)

Tag me when you're done and I'll merge it!

File: calculator.py [hunk 1]
```diff
@@ -38,6 +37,12 @@ def tan(self, x):
 
         return math.tan(x)
 
+    def sin(self, x):
```
Comment (GregHilston) [inline calculator.py]: Change this to:

`def sin(self, xInRadians):` 

and update the documentation and usage of `x`

**Comment by jkoors** [comment]
made the changes, check to see if all was correct on my end.  This has been very helpful, thanks!

**Comment by GregHilston** [comment]
Looks great @jkoors! I just updated the comment so no biggie.

Merging now! Excellent work man

**Pull request merged**